eth/logmonitor.go

```diff
@@ -0,0 +1,55 @@
+package eth
+
+import (
+	"context"
+
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/golang/glog"
+)
+
+type LogMonitor struct {
+	callbacks []func(j *Job)
+}
+
+func NewLogMonitor(eth LivepeerEthClient) *LogMonitor {
```

Comment:
Isn't most of the logic in this file already in `jobmanager.go`?

Reply:
Right... I couldn't find anywhere that actually invokes the JobManager... Maybe it's not being used right now?  Agree we should probably combine the two.  Let's talk on Monday.